src/BenchmarkDotNet/Toolchains/Mono/MonoAotToolchain.cs

```diff
@@ -15,7 +15,7 @@ public class MonoAotToolchain : Toolchain
         public static readonly IToolchain Instance = new MonoAotToolchain();
 
         [PublicAPI]
-        public MonoAotToolchain() : base("MonoAot", new Generator(), new MonoAotBuilder(), new Executor())
+        public MonoAotToolchain() : base("MonoAot", new Generator(), new MonoAotBuilder(), new Executor(), null)
```

Comment:
If not every toolchain needs an `ISdkProvider`, why is it part of the base toolchain?

Reply:
I forgot about this one. I am not familiar with Mono and just put `null` for the time being while I did more research. From what I found it is possible that Mono can use the .NET SDK since .NET 5. 

I will work on its implementation.

Reply:
Please do that in a separate PR if you're going to do it. But it's the same thing for in process toolchains and Roslyn toolchain.

Reply:
ah ok. I just committed some more changes but I will update my code.